include/binaryfusefilter.h

```diff
@@ -858,4 +860,91 @@ static inline bool binary_fuse8_deserialize(binary_fuse8_t * filter, const char
   return true;
 }
 
+// minimal bitfield implementation
+typedef uint8_t bitf_t;
```

Comment:
I am not exactly sure why you need a typedef here?

Reply:
If it is always going to be a uint8_t, then why bother with a typedef?

Reply:
Yes. This was a residue from [the more generic implementation that I copied](https://gitlab.com/uplex/varnish/slash/-/blob/master/src/bitsof.h?ref_type=heads#L29), but in here, using wider words makes no sense. Removing.